src/main/scala/ai/starlake/utils/Utils.scala

```diff
@@ -278,9 +279,17 @@ object Utils extends StrictLogging {
     result
   }
 
-  def parseJinjaTpl(templateContent: String, params: Map[String, Object]): String = {
-    val jinjava = new Jinjava()
-    jinjava.render(templateContent, params.asJava);
+  def parseJinjaTpl(templateContent: String, params: Map[String, Object])(implicit
+    settings: Settings
+  ): String = {
+    val config = JinjavaConfig
+      .newBuilder()
+      .withNestedInterpretationEnabled(false)
```

Comment:
What's the meaning of this ?
Are we relying on it ?

Reply:
If you have {{"{{ ds }}"}} then by default it will try to resolve ds var instead of just rendering it as "{{ ds }}". Therefore it would fail.

Reply:
Ok thanks 
Got it 👍